packages/kit/test/apps/basics/test/test.js

```diff
@@ -1798,3 +1798,37 @@ test.describe('Actions', () => {
 		await expect(page.locator('pre')).toHaveText(JSON.stringify({ result: 'foo' }));
 	});
 });
+
+test.describe('Cookies API', () => {
+	test('sanity check for cookies', async ({ page }) => {
+		await page.goto('/cookies');
+		const span = page.locator('#cookie-value');
+		expect(await span.innerText()).toContain('undefined');
+	});
+
+	test('set a cookie', async ({ page }) => {
+		await page.goto('/cookies/set');
+		const span = page.locator('#cookie-value');
+		expect(await span.innerText()).toContain('teapot');
+	});
+	test('delete a cookie', async ({ page }) => {
```

Comment:
```suggestion

	test('delete a cookie', async ({ page }) => {
```

Reply:
Ayayay, am I not formatting? I do pnpm format, I think.

Or is this something else?

Reply:
Oh, I see, you want a line between each test, probably fixed by prettier. I'll try to do that.